lib/modules/accountSwitcher.js

```diff
@@ -14,12 +14,7 @@ export const module: Module<*> = new Module('accountSwitcher');
 
 module.moduleName = 'accountSwitcherName';
 module.category = 'myAccountCategory';
-module.description = `
```

Comment:
I'll leave this as is, it'll be easy to find these later with a simple `Object.values(modules).filter(m => m.description.includes(' '))`; same with any module options